scmailclient/scmail.py

```diff
@@ -244,9 +236,12 @@ def register(ctx, fingerprint):
     type=str,
     help="The senders fingerprint.",
 )
-@click.password_option(
+@click.option(
     "--password",
     "-p",
+    type=str,
+    default="None",
+    hide_input=True,
```

Comment:
Any reason to use `hide_input` instead of `password_option`?

Reply:
No, thanks.